spotler/backend/plugin/src/main/kotlin/com/ritense/spotler/domain/SubmitMessage.kt

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2015-2024 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ritense.spotler.domain
+
+import com.ritense.valtimo.contract.mail.model.TemplatedMailMessage
+import org.apache.tika.Tika
+
+data class SubmitMessage(
+    val attachments: MutableList<Attachment> = mutableListOf(),
+    val data: MutableMap<String, Any>,
+    val flowSelector: String, //link to flow with template
```

Comment:
flow selector is this still called this way with spotler?

Reply:
Yeah a flow is a sequence of actions that every email will go through. like set subject, set sender